Fabric.Authorization.AccessControl/src/app/services/current-user.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@angular/core';
+import { FabricAuthUserService } from './fabric-auth-user.service';
+import { Observable } from '../../../node_modules/rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class CurrentUserService {
+  private permissions: Array<string> = [];
+  private initialized: boolean;
+
+  constructor(private authUserService: FabricAuthUserService) {
+  }
+
+  getPermissions(): Observable<string[]> {
+    if (!this.initialized) {
+      this.initialized = true;
+      return this.authUserService.getCurrentUserPermissions().do(userPermissionResponse => {
```

Comment:
Should we introduce a caching window? I don't think we want to cache these permissions forever...

Reply:
perhaps 2 minutes...